trace/opencensus/trace/exporters/transports/background_thread.py

```diff
@@ -107,14 +106,12 @@ def _thread_main(self):
                     'traceId': trace_id,
                     'spans': spans,
                 }
+
                 self.exporter.emit(spans_json)
 
             for _ in range(len(items)):
                 self._queue.task_done()
 
-            # Wait for a while before next export
-            time.sleep(_WAIT_PERIOD)
```

Comment:
Will removing the sleep make this into a busy loop and eat CPU?

Reply:
Yeah agree, changed to reduce the sleep time to 1 second. Just want to make sure the spans are sent within the grace period.